core/app/models/spree/fulfilment_changer.rb

```diff
@@ -20,15 +20,23 @@ class FulfilmentChanger
     include ActiveModel::Validations
 
     attr_accessor :current_shipment, :desired_shipment
-    attr_reader :variant, :quantity, :current_stock_location, :desired_stock_location
+    attr_reader :variant, :quantity, :current_stock_location, :desired_stock_location, :track_inventory
 
-    def initialize(current_shipment:, desired_shipment:, variant:, quantity:)
+    def initialize(current_shipment:, desired_shipment:, variant:, quantity:, track_inventory: nil)
```

Comment:
I'm thinking `nil` might be somehow a valid value for `Spree::Config.track_inventory_levels`, as it still evaluates to `false`. I'm thinking the default value for `track_inventory` may be a constant with an arbitrary value and use that instead of `nil`, so we're 100% sure we didn't receive that param from the caller. Something like what's being done [in this PR](https://github.com/solidusio/solidus/pull/4909/files) with `PAYMENT_NOT_PROVIDED`. WDYT?

Reply:
I thought people always need to pass something that is `true` or `false` explicitly here. Do you mean that someone might pass `nil` initializing this class, but they actually mean to pass false? I'm not sure when that could happen, but open to changing this if you have a concrete example.

Reply:
@kennyadsl looks great, thanks!